### PR TITLE
record: add get_values_for_schema

### PR DESCRIPTION
**Pull request by michamos**

### CODE REVIEW

File: inspire_utils/record.py [hunk 1]
```diff
@@ -80,3 +80,25 @@ def getitem(k, v, default):
...
+    Returns:
+        list: all values conforming to the given schema.
+
+    Example:
+        >>> elements = [
+        ...     {'schema': 'TWITTER', 'value': 's_w_hawking'},
+        ...     {'schema': 'WIKIPEDIA', 'value': 'Stephen_Hawking'}
+        ... ]
+        >>> get_values_for_schema(elements, 'TWITTER')
+        ['Stephen_Hawking']
```
Comment (iulianav) [inline inspire_utils/record.py]: Shouldn't it return ['s_w_hawking'] in this example?

Comment (michamos) [reply inspire_utils/record.py]: I fixed it

File: inspire_utils/record.py [hunk 2]
```diff
@@ -80,3 +80,25 @@ def getitem(k, v, default):
...
+
+    Example:
+        >>> elements = [
+        ...     {'schema': 'TWITTER', 'value': 's_w_hawking'},
+        ...     {'schema': 'WIKIPEDIA', 'value': 'Stephen_Hawking'}
+        ... ]
+        >>> get_values_for_schema(elements, 'TWITTER')
+        ['s_w_hawking']
+    """
+    return [element['value'] for element in elements if element['schema'] == schema]
```
Comment (puntonim) [inline inspire_utils/record.py]: why not a generator (`yield`)?

Comment (michamos) [reply inspire_utils/record.py]: @jacquerie said no :)

Comment (puntonim) [reply inspire_utils/record.py]: @jacquerie thinks small... Just kidding, I'm fine with this.

**Pull request merged**